typings/images.d.ts

```diff
@@ -13,7 +13,17 @@ declare module '*.gif' {
   export = value;
 }
 
+declare namespace SVGR {
```

Comment:
Not sure this belongs in `images.d.ts`...

Reply:
I'll take a look at this again. I had it in a separate file and for some reason I couldn't get it to work, but I feel like I must have been overlooking something obvious...

Reply:
I think I must have named the file `svgr.ts` instead of `svgr.d.ts` in my first try, because it all seemed to work beautifully when I tried again. Pushed up a fix.

Reply:
👍